src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/sonar-project.properties

```diff
@@ -0,0 +1,12 @@
+    sonar.projectKey=<copy-this-from-the-sonarqube-server-webpage>
+    sonar.projectName=<project-name>
```

Comment:
This - and projectVersion below - should be possible to populate automatically with the values from cookiecutter

Reply:
Added